problemtools/verifyproblem.py

```diff
@@ -145,7 +154,50 @@ def __str__(self):
     def matches_filter(self, filter_re):
         return filter_re.search(self.strip_path_prefix(self._base)) is not None
 
+    def set_symlinks(self):
+        if not os.path.islink(self.infile):
+            return
```

Comment:
Aren't these two lines redundant given the check below that `target != self.infile`?

Reply:
They are, but I think they add clarity that this only affects symlinks. OTOH, I guess the `target != self.infile` check is redundant since a symlink pointing to itself would create an unresolvable infinite loop.